pastis/matrix_generation/matrix_from_efields.py

```diff
@@ -166,12 +181,12 @@ def instantiate_simulator(self):
         """ Create a simulator object and save to self.simulator """
         raise NotImplementedError()
 
-    def calculate_ref_efield(self):
+    def calculate_ref_efield(self, norm_one_photon):
```

Comment:
You are passing a variable `norm_one_photon` to this method, but in the end, in line 189, you actually use the class attribute `self.norm_one_photon` instead of the variable, which means that the variable remains unused. Considering that the method `calculate_ref_efield()` is a class method required by `PastisMatrixEfields`, which is also used by RST, which does not have the option to normalize its outputs, I would remove the input variable to the method and keep using the class attribute. This would mean:
- [ ] remove the variable in the function signature here (line 184)
- [ ] remove the variable in the function signature for `calculate_ref_efield_wfs()` in line 210
- [ ] remove the variable in the function signature in lines 94 and 98
- [ ] remove the variable in the function calls in lines 57 and 58
In this way, you don't need to adjust this method inside of `MatrixEfieldRST` and can remove the question in the todo comment there.